torch_xla/__init__.py

```diff
@@ -1,10 +1,22 @@
 import logging
 import os
 import re
+import sys
 import tempfile
 import warnings
 
 import torch
+
+if not torch.cuda.is_available():
+  # Load _XLAC_cuda_functions to RTLD_GLOBAL, so that it can be used by _XLAC.
+  flags = sys.getdlopenflags()
+  sys.setdlopenflags(flags | os.RTLD_NOW | os.RTLD_GLOBAL)
+
+  import _XLAC_cuda_functions
+
+  # Then, restore the original flags.
+  sys.setdlopenflags(flags)
```

Comment:
Do you know why we need to restore the original flags?

Reply:
I'd guess that, in general, we don't really want to load things and make them available globally (i.e. some encapsulation for loaded functions).